src/compas_timber/solvers/__init__.py

```diff
@@ -0,0 +1,77 @@
+"""
+********************************************************************************
+solvers
+********************************************************************************
+
+.. currentmodule:: compas_timber.solvers
+
+.. rst-class:: lead
+
+This module contains pluggable functions for the sequence generating puzzle checker extensions.
+
+Pluggables
+==========
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    next_removable_part
+    create_dissassembly_sequence
+
+"""
+from compas.plugins import pluggable
+from compas.plugins import PluginNotInstalledError
+
+
+@pluggable(category="solvers")
+def next_removable_part(assembly, removed_part_ids, part_ids_to_remove, number_of_parts_to_remove, **kwargs):
```

Comment:
just a side thought: is there a way to put all these definitions of the pluggables in seperate files? They are all quite specific and I guess the `__init__` file can get quite messy soon.